src/components/dialogs/commons/modificationDialog.js

```diff
@@ -46,9 +46,24 @@ const ModificationDialog = ({
         closeAndClear(data, 'validateButtonClick');
     };
 
-    const handleValidationError = (errors) =>
+    const handleValidationError = (errors) => {
         onValidationError && onValidationError(errors);
 
+        // When scrolling to the field with focus, you can end up with the label not completely displayed
+        // We ensure that field with focus is displayed in the middle of the dialog
+        // Delay focusing to ensure it happens after the validation. without timout, document.activeElement will return the validation button.
+        setTimeout(() => {
```

Comment:
you should use the setTimeout function in a useCallback or a useEffect. Don't forget the return to clear the timeout